src/routes/api/installation/api-installation-sync-post.ts

```diff
@@ -22,7 +22,7 @@ export const ApiInstallationSyncPost = async (req: Request, res: Response): Prom
 			return;
 		}
 
-		await findOrStartSync(subscription, req.log, resetType);
+		await findOrStartSync(subscription, req.log, resetType, undefined);
```

Comment:
if you make `commitsFromDate` optional, passing `undefined` is not required.

Reply:
Per my comments above, making it explicit `undefined` is "better" in my mind.
As code is written only once, but read/run millions times, I prefer making it more explicit.